scripts/west_commands/runners/bossac.py

```diff
@@ -41,6 +43,8 @@ def do_add_parser(cls, parser):
                                  ' to a flash page boundary')
         parser.add_argument('--bossac-port', default='/dev/ttyACM0',
                             help='serial port to use, default is /dev/ttyACM0')
+        parser.add_argument('--speed', default='115200',
+                            help='serial port speed to use, default is 115200')
```

Comment:
Please use DEFAULT_BOSSAC_SPEED instead of hard-coding

Reply:
Fixed for SPEED and PORT.